ddcz/views.py

```diff
@@ -599,3 +602,28 @@ def tavern(request):
             "current_list_style": list_style,
         },
     )
+
+
+def sign_up(request):
+    form = SignUpForm()
+
+    if request.method == "POST" and request.POST["submit"]:
+        sgn = SignUpForm(request.POST)
+        if sgn.is_valid():
+            sgn.save()
+            return HttpResponseRedirect(reverse("ddcz:news"))
```

Comment:
Will this tell user that the registration was sent? If not, `message.info` is in order :)